detections/endpoint/windows_theme_command_execution.yml

```diff
@@ -0,0 +1,79 @@
+name: Windows Theme Command Execution
+id: 5f1c168e-118b-11ec-84ff-acde48001124
+version: 1
+date: '2023-09-25'
+author: Michael Haag, Gowthamaraj Rajendran, Splunk
+status: experimental
+type: TTP
+description: The vulnerability, referred to as ThemeBleed, is a high-severity issue with a score of 8.8. 
+  It allows remote attackers to execute code if the target user opens a malicious .THEME file crafted by the attacker. 
+  The exploit takes advantage of a race condition when handling .MSSTYLES files. Specifically, when a version number 999 is used, 
+  there is a discrepancy between the time a _vrf.dll signature is verified and when the library loads.
+  This allows an attacker to replace a verified DLL with a malicious one, leading to arbitrary code execution.
+data_source:
+- Sysmon
+search: '| tstats `security_content_summariesonly` count min(_time) as firstTime max(_time)
+  as lastTime from datamodel=Endpoint.Processes where 
+  (Processes.process="C:\\Windows\\system32\\rundll32.exe*themecpl.dll,OpenThemeAction*theme") OR (Processes.process="*rundll32.exe uxtheme.dll,#64 *.msstyles*") by Processes.dest Processes.user Processes.parent_process
```

Comment:
missing `Processes.parent_process_name` in the SPL, you can see its being used as an observable below.